src/integrations/cms.js

```diff
@@ -68,7 +68,12 @@ export const updateCmsData = async (
         resolveLinks: false
       };
 
-  const cmsData = await clientObj.sync(syncOpts);
+  let cmsData;
+  try {
+    cmsData = await clientObj.sync(syncOpts);
+  } catch (e) {
+    return { ...localCmsData, updated: false };
+  }
```

Comment:
I have assumed the try/catch pattern is ok here?

Reply:
Not sure if we want to apply the same pattern here as well? https://github.com/redbadger/pride-london-app/pull/357/files#diff-55210a6e2d724432b6b63cbc601e4479R42

Reply:
Yea, actually, we probably should, since we can recover from that failure